part3.asm

```diff
@@ -0,0 +1,40 @@
+# Author: Tam Duong
+# Assignment: PA1
+# Date: 2/11/15
+
+	.data
+
+OddSum:		.asciiz	"Sum of odd numbers between 1 and 15 inclusive: "	# headline for the result
+result:		.word 0:10		# array of length 10 to store each sum
+	
+	.text	
+
+	li $s0, 1	# $s0 = start = 1
+	li $s1, 15	# $s1 = end = 15
+	li $t0, 2	# $t0 = 2
+	la $s2, result	# $s2 = result index 
+	sw $zero, 0($s2)	# result[0] = 0 let say it start with 0
+	
+loop:	bgt $s0, $s1, exit	# start > end then exit loop
+	div $s0, $t0		# $s0 / 2 = start / 2
```

Comment:
Using div to check even/odd works well but simply incrementing your counter by 2 and adding it to your total would be more efficient.